public/index.css

```diff
@@ -117,6 +117,34 @@ body {
   padding: 0 3%;
 }
 
+.accordion {
+  display: flex;
+  flex-direction: column;
+  min-width: 350px;
+  padding: 0 10%;
+}
+
+.acc-item {
+  flex: 0 0 auto;
+  /* padding: 0 3rem; */
```

Comment:
I don't think we need left/right padding, but up/down padding can be nice for some spacing between the items. I think that `padding: 0.5rem 0;` looks nice. What do you think?

Reply:
Yes! They do need some space in betweens. Looks great!